src/services/EdFi.DataManagementService.Api.Tests.E2E/Hooks/SetupHooks.cs

```diff
@@ -39,6 +39,13 @@ TestLogger logger
                 logger.log.Debug("Using local environment, verify that it's correctly set.");
             }
 
+            string? dbEngine = _configuration["DatabaseEngine"];
+
+            if (dbEngine != null)
+            {
+                context.Database = await containers.SetupDataBase(dbEngine);
+            }
+
```

Comment:
I see `if (useTestContainers)` above on line 32.

Would it make sense to put your new code into that same branching statement, instead of being below the `else`?